processing/src/main/java/io/druid/segment/serde/DictionaryEncodedColumnPartSerde.java

```diff
@@ -349,13 +364,20 @@ public void read(ByteBuffer buffer, ColumnBuilder builder, ColumnConfig columnCo
           case UNCOMPRESSED_MULTI_VALUE: {
             return VSizeColumnarMultiInts.readFromByteBuffer(buffer);
           }
+          case UNCOMPRESSED_WITH_FLAGS: {
+            if (Feature.MULTI_VALUE.isSet(flags)) {
+              return VSizeColumnarMultiInts.readFromByteBuffer(buffer);
+            } else {
+              throw new IAE("Unrecognized multi-value flag[%d] for version[%s]", flags, version);
```

Comment:
missing multi value V3 handling ?

Reply:
This is missing on purpose; from the code it looks like MULTI_VALUE_V3 is only supported for compressed formats.

Reply:
please add a comment.

Reply:
Will add a comment